pkg/actors/actors.go

```diff
@@ -1796,7 +1796,7 @@ func (a *actorsRuntime) doDeleteReminder(ctx context.Context, req *DeleteReminde
 
 	stop, exists := a.activeReminders.Load(reminderKey)
 	if exists {
-		log.Infof("Found reminder with key: %v. Deleting reminder", reminderKey)
+		log.Debugf("Found reminder with key: %v. Deleting reminder", reminderKey)
```

Comment:
This Info log predates my involvement in the project (and predates other people I've asked). I don't really understand why it was Info, but it feels like a good candidate for Debug given that it's written every time a reminder expires or gets deleted.

Reply:
Can the key contain any PII? Or is it always a random identifier?

Reply:
For normal actors, the reminder names are controlled by the developer and can be anything they want. For internal actors used by workflows, we control the reminder names and there is no risk of PII.

Reply:
> For normal actors, the reminder names are controlled by the developer and can be anything they want.

Then consider not logging the name as it may contain PII.

Reply:
Logging reminder names is unrelated to Dapr Workflow. Logging of reminder names has been in the Dapr codebase since before I got involved. This is also not the only place we log reminder names. Can we make this bigger change as part of a separate PR? This PR is simply about reducing distracting log noise.